### PR TITLE
Fixed typo 

### LINKED ISSUE #98: cEP-0002.md Correct sentence
`No longer will the local bears be dependent on global bears to finish`, `local` and `global` are interchanged

### PR DESCRIPTION
**Pull request by kaustumbh7**
cEP-0002.md: Correct sentence 

This corrects the phrasing mistake and interchanges
the words 'local' and 'global'.

Closes #98

### CODE REVIEW

**Review by shreyateeza — APPROVED** [review]
LGTM!

**Review by newbazz — CHANGES_REQUESTED** [review]
Change `Fixes` to `fixes` in the commit body ;)

**Comment by kaustumbh7** [comment]
Ohh! Sorry my bad! ;)

**Comment by newbazz** [comment]
Make changes and we will have it merged by tonight ;P

**Comment by kaustumbh7** [comment]
Yeah sure! :)

**Comment by kaustumbh7** [comment]
Is it fine now?

**Review by newbazz — APPROVED** [review]
LGTM 
cc @Makman2

**Comment by Makman2** [comment]
Imperative tense please for shortlog:
--> Fix typo

Also: This is not a real typo, but just a phrasing mistake. Could you maybe rephrase your commit message to resemble this?

**Comment by Makman2** [comment]
unack a1d90a3

**Comment by kaustumbh7** [comment]
@Makman2  Is it ok now?

**Comment by ishanSrt** [comment]
You are right this is not a typo. I wrote the issue incorrectly 😛. it would be better to remove the tag bug and the ask @kaustumbh7 to change the commit message accordingly. I will edit the issue 
cc @Makman2

**Comment by kaustumbh7** [comment]
@ishanSrt  Is it fine now?

**Comment by ishanSrt** [comment]
Use `Closes` instead of `Fixes` since this won't remain a bug or a typo 👍

**Comment by kaustumbh7** [comment]
@ishanSrt Done! :)

**Review by ishanSrt — APPROVED** [review]
@Makman2 ready to merge. please make the required changes on the issue https://github.com/coala/cEPs/pull/99#issuecomment-357419093

**Comment by Makman2** [comment]
ack 9f4b596

**Comment by Makman2** [comment]
@gitmate-bot ff
